_posts/2024-02-11-ocpv-migration.md

```diff
@@ -0,0 +1,226 @@
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/6.png)
+
+Once there enter the following command:
+
+    echo | openssl s_client -connect localhost:443 2>/dev/null | openssl x509 -noout -fingerprint -sha1
+
+This will output the SHA1 fingerprint:
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/8.png)
+
+Your provider is now complete.
+
+### NetworkMap
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/10.png)
+
+Click on Create NetworkMap. Use a sensible name for the mapping name, pick your source and destination providers (host will always be the destination name for openshift...it can't be changed), then finally map the VMware networks to the openshift networks.
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/11.png)
+
+NetworkMapping is complete.
+
+### StorageMap
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/12.png)
+
+Click Create StorageMap. Use a sensible name for the mapping name, pick your source and destination providers, then finally map the ESXi data stores to the openshift CSI storage.
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/13.png)
+
+StorageMapping is complete.
+
+### Setting up the Plan
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/16.png)
+
+Click Create Plan. Give the plan a name, select the source and target providers and select the namespace you wish to migrate the VM into. It's best to have this setup prior to creation of the plan:
+
+    oc create ns <vm namespace name>
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/17.png)
+
+Click next and select the correct VMware infrastructure for the source VM's:
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/18.png)
+
+Select the VM you wish to migrate. Note that MTV will warn you of any issues that will prevent machines from being migrated. The most obvious of which is that the machine name cannot contain capital letters. Select the VM or VM's you wish to migrate then click next.
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/19.png)
+
+Select the exiting network mapping you made previously, then click next:
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/23.png)
+ 
+ Select the existing storage mapping you made previously, then click next:
+ 
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/24.png)
+
+Select the migration type you wish to use...either Warm (online) or cold (offline), then click next:
+
+![enter image description here](/images/posts/2024-02-11-ocpv-migration/25.png)
+
+Note that for warm migrations to work you will need to enable **change block tracking** for each VM. This can be accomplished by setting the option in the advanced settings per VM. This can **only** be done when the VM is powered off:
```

Comment:
```suggestion
{: .alert-info }
Note that for warm migrations to work you will need to enable **change block tracking** for each VM. This can be accomplished by setting the option in the advanced settings per VM. This can **only** be done when the VM is powered off:
```